plugins/modules/docker_compose.py

```diff
@@ -214,7 +214,7 @@
         var: output
 
     - ansible.builtin.assert:
-        that: "not output.changed "
+        that: "not output.changed"
```

Comment:
TBH the quotes are not necessary either, and I'd leave them away:
```suggestion
        that: not output.changed
```